python/notebooks/05-using-the-planner.ipynb

```diff
@@ -53,8 +53,8 @@
     "else:\n",
     "    api_key, org_id = sk.openai_settings_from_dot_env()\n",
     "    kernel.add_chat_service(\n",
-    "        \"gpt-3.5\",\n",
-    "        OpenAIChatCompletion(ai_model_id=\"gpt-3.5-turbo\", api_key=api_key, org_id=org_id),\n",
+    "        \"gpt-4\",\n",
+    "        OpenAIChatCompletion(ai_model_id=\"gpt-4\", api_key=api_key, org_id=org_id),\n",
```

Comment:
why was this changed?

Reply:
While running that notebook with gpt-3.5-turbo, it threw an error in the last cell that the number of tokens (both input + completion) exceeded the allowed amount for that model. It seems I just need to set it to `gpt-3.5-turbo-1106` which has a larger context window.